src/simplicial-hott/04-extension-types.rzk.md

```diff
@@ -297,6 +297,19 @@ cases an extension type to a function type.
             is-contr-based-paths (A t ) ((ext-projection-temp) t))
       ( \ t → (a t , refl) )
 
+#define is-contr-ext-codomain-based-paths uses (weak-ext-ext f) 
+    : is-contr 
+        ((t : ψ ) → (Σ (y : A t) ,
+                    (y = (ext-projection-temp) t))[ϕ t ↦ (a t , refl)])
```

Comment:
I would probably reformat it like this:
```suggestion
        ( (t : ψ) → (Σ (y : A t) , y = ext-projection-temp t)
                    [ ϕ t ↦ (a t , refl) ])
```

@fredrik-bakke what do you think?

Reply:
To avoid excessively indented code, one principle that was proposed was to always break lines at the earliest reasonable point. Here, that would be right after the first arrow. I.e.
```suggestion
        ( ( t : ψ) →
          ( Σ (y : A t) , (y = ext-projection-temp t))
          [ ϕ t ↦ (a t , refl)])
```
Notice how this principle leads to a much less awkward placement of the last line.